### PR TITLE
fix bulk cell edit undo/redo selections

### PR DESCRIPTION
**Pull request by rebornix**

### CODE REVIEW

File: src/vs/workbench/contrib/bulkEdit/browser/bulkCellEdits.ts [hunk 1]
```diff
@@ -73,7 +76,13 @@ export class BulkCellEdits {
 			// apply edits
 			const edits = group.map(entry => entry.cellEdit);
 			const computeUndo = !ref.object.isReadonly();
-			ref.object.notebook.applyEdits(edits, true, undefined, () => undefined, this._undoRedoGroup, computeUndo);
+			const editor = getNotebookEditorFromEditorPane(this._editorService.activeEditorPane);
+			const initialSelectionState: ISelectionState | undefined = editor?.textModel?.uri.toString() === ref.object.notebook.uri.toString() ? {
```
Comment (Tyriar) [inline src/vs/workbench/contrib/bulkEdit/browser/bulkCellEdits.ts:80]: I just fixed an issue using this uri.toString around Windows drive letter yesterday. You might want to do something like this? https://github.com/microsoft/vscode/pull/169064

**Review by Tyriar — DISMISSED** [review]

File: src/vs/workbench/contrib/bulkEdit/browser/bulkCellEdits.ts (hunk 1 above)
Comment (rebornix) [reply src/vs/workbench/contrib/bulkEdit/browser/bulkCellEdits.ts:80]: Checking `fsPath` is not sufficient for resources, right (e.g, timeline view can open two documents with same fsPath). We are using `toString` for document uri comparison in notebook land and maybe we should revisit this but in this PR let's keep it consistent.

Comment (joyceerhl) [reply src/vs/workbench/contrib/bulkEdit/browser/bulkCellEdits.ts:80]: Also `fsPath` is only defined for resources that exist on disk, so using `fsPath` might break notebooks in vscode.dev?

**Review by joyceerhl — APPROVED** [review]

File: src/vs/workbench/contrib/bulkEdit/browser/bulkCellEdits.ts (hunk 1 above)
Comment (Tyriar) [reply src/vs/workbench/contrib/bulkEdit/browser/bulkCellEdits.ts:80]: @rebornix I still mean use toString(), but also check `fileService.hasCapability(uri, FileSystemProviderCapabilities.PathCaseSensitive)`

**Pull request merged**